packages/crayons-extended/platform-ui/fileutil.js

```diff
@@ -0,0 +1,2 @@
+const prependFile = require('prepend-file');
```

Comment:
both packages has this fileutil.js. can we move these common things outside of both packages and import from one common place ?

Reply:
Moved them to scripts folder at the root